### PR TITLE
Use an existing threshold for filtering senstivity of injection over voltage and voltage over injection

### PR DESCRIPTION
**Pull request by vidaldid-rte**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**
No

**What kind of change does this PR introduce?**
New sensitivities added in https://github.com/powsybl/powsybl-open-loadflow/pull/992 and https://github.com/powsybl/powsybl-open-loadflow/pull/987 are not filtered.
This PR reuses the existing threshold flowVoltageSensitivityValueThreshold to filter them.
We consider the other order of magnitude of these sensitivities is  similar to other sensitivities using this threshold.
(for instance this parameter is used for V to reactive power in a branch).

**What is the current behavior?**
The new sensitivies are not filtered

**What is the new behavior (if this is a feature change)?**
The new sensitivies are filtered with the flowVoltageSensitivityValueThreshold

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [X] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**
Users using the sensitivity BUS_VOLTAGE over BUS_REACTIVE_POWER, or INJECTION_REACTIVE_OVER over BUS_VOLTAGE should be aware that the sensitivities are now filtered by the flowVoltageSensitivityValueThreshold threshold.

**Other information**:

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java [hunk 1]
```diff
@@ -1253,6 +1253,10 @@ protected static boolean filterSensitivityValue(double value, SensitivityVariabl
         switch (variable) {
             case INJECTION_ACTIVE_POWER, HVDC_LINE_ACTIVE_POWER:
                 return isFlowFunction(function) && Math.abs(value) < parameters.getFlowFlowSensitivityValueThreshold();
+            case INJECTION_REACTIVE_POWER:
+                // We consider that the magnitude change of reactive injection to V is of the same magnitude as other usqges of FlowVoltage threshold
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java:1257]: typo: usages

File: src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java [hunk 2]
```diff
@@ -1265,7 +1269,9 @@ protected static boolean filterSensitivityValue(double value, SensitivityVariabl
     protected static boolean filterBusTargetVoltageVariable(double value, SensitivityFunctionType function,
                                                             SensitivityAnalysisParameters parameters) {
         return switch (function) {
-            case BRANCH_CURRENT_1, BRANCH_CURRENT_2, BRANCH_CURRENT_3 -> Math.abs(value) < parameters.getFlowVoltageSensitivityValueThreshold();
+            // We consider that the Voltage change to current is approximately of the same magnitude as the voltage change to reactive power
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java:1272]: typo: Voltage -> voltage

File: src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java (hunk 1 above)
Comment (vidaldid-rte) [reply src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java]: Done

File: src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java (hunk 2 above)
Comment (vidaldid-rte) [reply src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java]: Done

**Review by annetill — APPROVED** [review]

**Pull request merged**